.github/workflows/end2end-test.yml

```diff
@@ -36,10 +36,6 @@ jobs:
               run: |
                   npm run wp-env start
 
-            - name: Running the tests
-              run: |
-                  npx wp-scripts test-e2e --config=./packages/e2e-tests/jest.config.js --cacheDirectory="$HOME/.jest-cache"
-
```

Comment:
I'm so happy now! 🍾 🥳 🕺

Reply:
I've not removed it entirely. Requires some settings updates for the repo. I'll do a complete clean-up follow-up PR.